.github/workflows/ci.yml

```diff
@@ -78,17 +78,8 @@ jobs:
       fail-fast: false
       matrix:
         php: ['7.4', '8.0', '8.1', '8.2']
-        symfony_version: ['5.4.*', '6.2.*', '6.3.*']
+        symfony_version: ['5.4.*', '6.4.*', '7.0.*']
         dependencies: ['--prefer-lowest', '--prefer-dist']
-        exclude:
-          - php: '7.4'
-            symfony_version: '6.2.*'
-          - php: '7.4'
-            symfony_version: '6.3.*'
-          - php: '8.0'
-            symfony_version: '6.2.*'
-          - php: '8.0'
-            symfony_version: '6.3.*'
```

Comment:
You need to keep exclude for SF 6.4 and SF 7, because 
- SF 6.4 doesn't work on php 7.4 and 8.0
- SF 7 doesn't work on php 7.4, 8.0 and 8.1